build/embed_ohis.mk

```diff
@@ -4,20 +4,27 @@ get-nri-version			= $(shell awk -F, '/^$(1),/ {print $$2}' ${NRI_INTEGRATIONS_FI
 NRI_PKG_DIR				?= $(PKG_DIR)
 PKG_DIR_BIN_OHIS    	?= $(NRI_PKG_DIR)/var/db/newrelic-infra/newrelic-integrations/bin
 
+# Default OHI ARCH and OS
+OHI_ARCH ?= amd64
+OHI_OS   ?= linux
+
 # nri-docker
 NRI_DOCKER_VERSION ?= $(call get-nri-version,nri-docker)
-NRI_DOCKER_ARCH    ?= amd64
-NRI_DOCKER_URL     ?= https://download.newrelic.com/infrastructure_agent/binaries/linux/$(NRI_DOCKER_ARCH)/nri-docker_linux_$(NRI_DOCKER_VERSION)_$(NRI_DOCKER_ARCH).tar.gz
+NRI_DOCKER_ARCH    ?= $(OHI_ARCH)
+NRI_DOCKER_OS      ?= $(OHI_OS)
+NRI_DOCKER_URL     ?= https://download.newrelic.com/infrastructure_agent/binaries/$(NRI_DOCKER_OS)/$(NRI_DOCKER_ARCH)/nri-docker_$(NRI_DOCKER_OS)_$(NRI_DOCKER_VERSION)_$(NRI_DOCKER_ARCH).tar.gz
 
 # flex
 NRI_FLEX_VERSION   ?= $(call get-nri-version,nri-flex)
-NRI_FLEX_OS        ?= Linux
-NRI_FLEX_URL       ?= https://github.com/newrelic/nri-flex/releases/download/v$(NRI_FLEX_VERSION)/nri-flex_$(NRI_FLEX_VERSION)_$(NRI_FLEX_OS)_x86_64.tar.gz
+NRI_FLEX_ARCH      ?= $(OHI_ARCH)
+NRI_FLEX_OS        ?= $(OHI_OS)
+NRI_FLEX_URL       ?= https://github.com/newrelic/nri-flex/releases/download/v$(NRI_FLEX_VERSION)/nri-flex_$(NRI_FLEX_OS)_$(NRI_FLEX_VERSION)_$(NRI_FLEX_ARCH).tar.gz
```

Comment:
did we change this URL schema from _linux_x86_64 to _amd64?

Reply:
Yeah, I updated the names of the Flex artifacts to match the rest of the OHIs... Except on doing this I realised that POMI does not match the rest of our OHIs 😕